requirements.txt

```diff
@@ -17,18 +17,18 @@ numpy>=1.18.1
 pandas>=1.1.5
 pillow>=6.2.0; platform_system == "Linux" or platform_system == "Darwin" or platform_system=='Windows'
 psutil>=5.6.6
-py~=1.10.0; platform_system != "Windows" or python_version <= "3.10"
-py==1.11.0; platform_system == "Windows" and python_version >= "3.11"
+py~=1.10.0; python_version <= "3.9"
+py==1.11.0; python_version >= "3.10"
```

Comment:
Isn't this removing the newly added change at https://github.com/wazuh/wazuh-jenkins/issues/5407?

Reply:
Not exactly, it is expanding it outside Windows. The other PR was only for Windows but this one has a broader scope and has been investigated as so.